### PR TITLE
DOC-630 "upgrade" FAQ article around OSS compatibility into full article

### PR DESCRIPTION
**Pull request by kaitlynmichael**
Some wording changes in the compatibility article. I removed the embed on the FAQ and linked it to the compatibility article instead.

### CODE REVIEW

File: content/rs/concepts/compatibility.md [hunk 1]
```diff
@@ -1,49 +1,40 @@
...
+- Shared databases aren't supported in Redis Software, because of the potential for
+    negative impact on performance. We recommend using
+    dedicated databases instead. The following commands are
+    blocked and produce an error when executed on a dedicated database:
     - MOVE
     - SELECT
-- Because data persistence and backups are managed from RS's
-    management UI, the following commands are blocked:
+- Data persistence, backups, and access controls are managed from the Redis Software
+    admin console. The following commands are blocked:
```
Comment (lanceleonard) [inline content/rs/concepts/compatibility.md:26]: Is this one thought or two?  Kind of reads like it's a new thought.  If so, it should be a separate bullet.

File: content/rs/concepts/compatibility.md [hunk 2]
```diff
@@ -1,49 +1,40 @@
...
 the open source's specifications and makes every effort to update
-RS with the latest version of Redis.
+Redis Enterprise Software with the latest version of open source Redis.
 
 ## Redis commands
 
-Any standard Redis client can be used with RS. That said, some of
-Redis functionality is not applicable in the context of RS, as
-follows:
+Any standard Redis client can be used with Redis Software. There is some
```
Comment (lanceleonard) [inline content/rs/concepts/compatibility.md:16]: Consider separating into different pargraphs.

File: content/rs/concepts/compatibility.md [hunk 3]
```diff
@@ -81,7 +69,6 @@ follows:
     - zset-max-ziplist-entries
     - zset-max-ziplist-value
 
-## Compatibility with Open Source Redis cluster
+## Compatibility with open source Redis Cluster API
 
-RS supports [Redis OSS cluster
-protocol]({{< relref "/rs/concepts/data-access/oss-cluster-api.md" >}}) if it is enabled for a database.
+Redis Enterprise Software supports [Redis OSS Cluster API]({{< relref "/rs/concepts/data-access/oss-cluster-api.md" >}}) if it is enabled for a database. For more information, see [Using the OSS Cluster API]({{< relref "/rs/administering/designing-production/networking/using-oss-cluster-api.md" >}})
```
Comment (lanceleonard) [inline content/rs/concepts/compatibility.md:74]: AFAICT, "OSS" hasn't been defined in this article.  Should it be?

**Review by lanceleonard — APPROVED** [review]
Consider these thoughts; otherwise LGTM.

File: content/rs/concepts/compatibility.md (hunk 3 above)
Comment (kaitlynmichael) [reply content/rs/concepts/compatibility.md:76]: Added OSS definition to first use of "open source Redis"

**Pull request merged**